dvc/commands/experiments/show.py

```diff
@@ -115,6 +115,7 @@ def _collect_rows(
                 )
             elif parent_rev == base_rev:
                 typ = "checkpoint_base"
+                parent = parent_rev[:7]
```

Comment:
How'd this affect the "rich" table? I am not sure about expected behaviour there. Is this only for CSV?

When using the test data from below, I see the table difference as:

###### on `HEAD`
```
  Experiment                Created               epoch   mult   start   EPOCHS   STEP
 ──────────────────────────────────────────────────────────────────────────────────────
  workspace                                           1      1   0       2        1
  main                      2021-03-10T14:05:52                  0       2        1
  │ ╓ 0a2b737 [exp-e44da]   2021-11-18T20:30:27       1      1   0       2        1
  ├─╨ 8acf8ec (0fd92c2)     2021-11-18T20:30:26       0      0   0       2        1
 ──────────────────────────────────────────────────────────────────────────────────────
```

###### on `main`
```
  Experiment                Created               epoch   mult   start   EPOCHS   STEP
 ──────────────────────────────────────────────────────────────────────────────────────
  workspace                                           1      1   0       2        1
  main                      2021-03-10T14:05:52                  0       2        1
  │ ╓ 0a2b737 [exp-e44da]   2021-11-18T20:30:27       1      1   0       2        1
  ├─╨ 8acf8ec               2021-11-18T20:30:26       0      0   0       2        1
 ──────────────────────────────────────────────────────────────────────────────────────
```

Reply:
I think `parent` should stay `None` here (for both CSV and the rich table). It's only supposed to be set for checkpoints that branch off of an existing checkpoint